sql/_units.sql

```diff
@@ -110,6 +110,18 @@ CORRECTIONS
 */
 CALL apply_correction('_UNITS_devdb', '_manual_corrections', 'resid_flag');
```

Comment:
corrections is still applying to '_UNITS_devdb'

Reply:
nice catch, making this change now